src/js/reducers/mainReducer.js

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
 
+import documentsReducer from './documentsReducer';
 import documentReducer from './documentReducer';
 
 const rootReducer = combineReducers({
+  documentsReducer,
```

Comment:
I think that you can store those two reducers into one file - they have same topic.